eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java

```diff
@@ -222,9 +230,9 @@ private Collection<String> probes(final Path file)
      * @todo #1395:30min Need to add the logic of "hasReservedChars" method to
      *  add-probes.xsl". After that, the method in this class need to be removed.
      */
-    private static boolean hasReservedChars(final String str) {
+    private static boolean notContainsReservedChars(final String str) {
```

Comment:
@volodya-lombrozo Clearer name is `missesReservedChars`.

Reply:
@mximp Done